src/rust/plugin-bootstrap/plugin.service

```diff
@@ -0,0 +1,54 @@
+# Contents of /etc/systemd/system/plugin.service
+[Unit]
+Description=Plugin
+After=plugin-bootstrap-init.service
+
+[Service]
+Type=simple
+Restart=always
+ExecStart=/usr/local/bin/grapl-plugin
+
+EnvironmentFile=/etc/systemd/system/plugin.service.d/override.conf
+
+# All plugins should listen on port 5555 internally
+SocketBindAllow=tcp:5555
+SocketBindDeny=any
+
+RuntimeDirectory=plugin        # adds /var/run/plugin
+StateDirectory=plugin          # adds /var/lib/plugin
+CacheDirectory=plugin          # adds /var/cache/plugin
+LogsDirectory=plugin           # adds /var/log/plugin
+LoadCredential=CLIENT_CERTIFICATE:/etc/ssl/private/plugin-client-cert.pem
```

Comment:
Looks like `CLIENT_CERTIFICATE` is never used?

Reply:
It'll be used by the plugin code, which isn't part of this PR.